sdk/go/common/resource/plugin/rpc.go

```diff
@@ -424,7 +424,14 @@ func UnmarshalPropertyValue(key resource.PropertyKey, v *structpb.Value,
 				case idProp.IsString():
 					id = idProp.StringValue()
 				case idProp.IsComputed():
-					// Leave the ID empty to indicate that it is unknown.
+				// Leave the ID empty to indicate that it is unknown.
```

Comment:
I think we want to keep the indentation as it was. Did `gofmt` remove the indentation?

Reply:
Ah no, just leftovers from when working this out. I've changed It back.